ParseLoginUI/build.gradle

```diff
@@ -1,13 +1,13 @@
 apply plugin: 'android-library'
 
 dependencies {
-    compile rootProject.ext.bolts
-    compile rootProject.ext.androidSupport
-    compile rootProject.ext.parse
+    compile 'com.parse.bolts:bolts-android:1.2.1'
```

Comment:
Because if developers import the ParseLoginUI to their project, in most of the cases, they do not have ext variable in their root gradle. So they have to manually copy them, which is not so convenient.